packages/core/src/Stamp/Stamp.js

```diff
@@ -66,8 +66,8 @@ const Stamp = styled.div`
   color: ${getPaletteColor('base')};
 
   ${space} ${fontSize};
-  ${applySizes(sizes)}
-  ${applyVariations('Stamp', variations)}
+  ${applySizes(sizes)};
+  ${applyVariations('Stamp', variations)};
 `
 
 Stamp.displayName = 'Stamp'
```

Comment:
JW, do we need to make any changes to the `sizes` prop type, or add any additional values?